codegen-client/src/test/kotlin/software/amazon/smithy/rust/codegen/client/smithy/transformers/RemoveEventStreamOperationsTest.kt

```diff
@@ -63,12 +89,38 @@ internal class RemoveEventStreamOperationsTest {
     fun `keep event stream ops from services that are in the allow list`() {
         val transformed =
             RemoveEventStreamOperations.transform(
-                model,
+                model(protocol = "awsJson1_0", rest = false),
                 testClientRustSettings(
+                    service = ShapeId.from("test#TestService"),
                     codegenConfig = ClientCodegenConfig(eventStreamAllowList = setOf("test-module")),
                 ),
             )
         transformed.expectShape(ShapeId.from("test#BlobStream"))
         transformed.getShape(ShapeId.from("test#EventStream")) shouldNotBe Optional.empty<Shape>()
     }
+
+    @Test
+    fun `keep event stream ops for rest services`() {
+        var transformed =
+            RemoveEventStreamOperations.transform(
+                model(protocol = "restJson1", rest = true),
+                testClientRustSettings(
+                    service = ShapeId.from("test#TestService"),
+                    codegenConfig = ClientCodegenConfig(eventStreamAllowList = setOf()),
+                ),
+            )
+        transformed.expectShape(ShapeId.from("test#BlobStream"))
+        transformed.getShape(ShapeId.from("test#EventStream")) shouldNotBe Optional.empty<Shape>()
+
+        transformed =
+            RemoveEventStreamOperations.transform(
+                model(protocol = "restXml", rest = true),
+                testClientRustSettings(
+                    service = ShapeId.from("test#TestService"),
+                    codegenConfig = ClientCodegenConfig(eventStreamAllowList = setOf()),
+                ),
+            )
+        transformed.expectShape(ShapeId.from("test#BlobStream"))
+        transformed.getShape(ShapeId.from("test#EventStream")) shouldNotBe Optional.empty<Shape>()
+    }
```

Comment:
Perhaps worth it to test that non-REST services with event stream ops have them removed.

Reply:
Looks like those tests exist already by `remove event stream ops from services that are not in the allow list`?

Reply:
The tests above this are covering that.